andela/ah-shakas-frontend
# #160577430 Setup an empty react project

#### What does this PR do?
- It sets up an empty React project.
- Initializes the default `.env` file to be used for environment variables:

#### Description of Task to be completed?
- Simply use `npx create-react-app .` on the existing repository. 
- Create different environment variable files for different environments.

#### How should this be manually tested?
- Clone the github repository `git clone https://github.com/andela/ah-shakas-frontend` on your machine.
- Change your working directory to the project folder `cd ah-shakas-frontend`
- Install the required dependencies using `yarn install`
- Run the local development server using `yarn start` and you should get the following message:
  ```
  Compiled successfully!

  You can now view ah-shakas-frontend in the browser.
  
    Local:            http://localhost:3000/
    On Your Network:  http://10.10.160.30:3000/
  
  Note that the development build is not optimized.
  To create a production build, use yarn build.
  ```

#### Any background context you want to provide?
  Create React App now supports the use of different environment variables files out of the box. For 
  more information click on this link: [here](https://medium.com/@tacomanator/environments-with-create-react-app-7b645312c09d).

  Therefore, other than creating the relevant files, nothing more needs to be done to configure their support.

#### What are the relevant pivotal tracker stories?
  [#160577430](https://www.pivotaltracker.com/story/show/160577430)

#### Screenshots (if appropriate)
  N/A
